src/applications/disability-benefits/all-claims/components/NewDisability.jsx

```diff
@@ -2,17 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { NULL_CONDITION_STRING } from '../constants';
-import { getShowAddDisabilitiesEnhancement } from '../content/addDisabilities';
 
 export default function NewDisability({ formData }) {
-  const showEnhancement = getShowAddDisabilitiesEnhancement();
-
   return (
-    <div
-      className={`${
-        showEnhancement ? 'vads-u-margin-left--1' : 'vads-u-flex--fill'
-      } vads-u-padding-right--2 word-break capitalize-first-letter`}
-    >
+    // TODO: Remove vads-u-flex--fill when allClaimsAddDisabilitiesEnhancement feature flag is removed
```

Comment:
Just curious, what's the diff here? Platform array field has diff margins or something?

Reply:
Thank you!